packages/web-components/src/player-component/player.class.ts

```diff
@@ -438,6 +438,46 @@ export class PlayerWrapper {
         }
     }
 
+    private disableNextSegmentButton(disable: boolean) {
+        for (const element of this.controls?.elements_) {
+            if (element?.isNextSegmentButton) {
+                element.disableNextSegmentButton(disable);
+            }
+        }
+    }
+
+    private disablePrevSegmentButton(disable: boolean) {
+        for (const element of this.controls?.elements_) {
+            if (element?.isPrevSegmentButton) {
+                element.disablePrevSegmentButton(disable);
+            }
+        }
+    }
+
+    private updateNextSegmentButton() {
```

Comment:
these are too small and not being used anywhere else and called one after another,, why not merge them to single function called updatePrevNextSegmentButtonsDisabledState ?

Reply:
The two functions are combined to updatePrevNextSegmentButtonsDisabledState.